docs/releases/202409.mdx

```diff
@@ -0,0 +1,18 @@
+---
+id: 202409
+title: September 2024
+displayed_sidebar: releaseSideBar
+---
+
+### September 5 2024
+
+**Features**
+- **Agent**: A new agent type is available that supports more detailed file inspection and manipulation prior to uploading to the Ganymede cloud
+
+
+**Improvements**
+- **UI**: The environment Secrets table now supports sorting by Name and Updated Date
+- **UI**: The Agent log date filter now reflects the dates where logs are available
```

Comment:
Did you want to mention the fact that users can copy/paste the connection and agent configurations from the UI now?

Reply:
Only proposed since you mentioned it was useful for debugging and communication to the Ganymede team.

Reply:
Yes we should add that